cmd/kinflate/demos/helloWorld.md

```diff
@@ -285,6 +311,112 @@ To deploy, pipe the above commands to kubectl apply:
 >    kubectl apply -f -
 > ```
 
+## Rolling updates
```

Comment:
I suggest to change this line to `Rolling updates for ConfigMaps`. Overall looks good to me. Feel free to merge the change.

Reply:
made that change, thanks for the review!